packages/video_player/video_player/lib/video_player.dart

```diff
@@ -243,6 +244,27 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
         package = null,
         super(VideoPlayerValue(duration: Duration.zero));
 
+  /// Constructs a [VideoPlayerController] playing a video from obtained from
+  /// the network.
+  ///
+  /// The URI for the video is given by the [dataSource] argument and must not be
+  /// null.
```

Comment:
Remove "and must not be null", since we no longer need comments like that given Dart null safety.